src/main/java/org/cascadebot/cascadebot/commands/fun/RandomNumberCommand.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.fun;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ICommandMain;
+import org.cascadebot.cascadebot.commandmeta.Module;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import java.io.IOException;
+import java.util.Random;
+
+public class RandomNumberCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        Random randomObj = new Random();
+        if (context.getArgs().length > 0) {
+            String argRaw = context.getArg(0);
+            if (argRaw.matches("[0-9]+") == false) {
+                context.getTypedMessaging().replyInfo("Please provide numbers only");
```

Comment:
This should be a danger message, i.e. `replyDanger()`

Reply:
uhm, Is this resolved?

Reply:
No it's not

Reply:
it is

Reply:
Nevermind